internal/backend/crypto/gpg/cli/gpg_windows_test.go

```diff
@@ -17,7 +17,9 @@ func TestEncrypt(t *testing.T) {
 	g.binary = "rundll32"
 
 	_, err := g.Encrypt(ctx, []byte("foo"), nil)
-	require.NoError(t, err)
+
+	// No recipients are configured so it will fail
+	require.Error(t, err)
```

Comment:
We might want to add a test with a recipient now to have a "positive" test, but that can be opened as an issue and done in another PR I think,

Reply:
> We might want to add a test with a recipient now to have a "positive" test, but that can be opened as an issue and done in another PR I think,

I wanted to do that, but I didn't find a method to add a recipient to that context / gpg in that test...